src/screens/send/components/address-book/address-book.tsx

```diff
@@ -43,8 +41,7 @@ export interface IExternalProps {
 
 export const mapStateToProps = (state: IReduxState, ownprops: IExternalProps) => {
     return {
-        contacts: selectContacts(state, ownprops.blockchain),
-        hints: state.app.hints
+        contacts: selectContacts(state, ownprops.blockchain)
```

Comment:
Why did you removed the hints from here?

Reply:
That was my understanding of the issue. "All the cards that have hidden menu..."

Reply:
And the hints were not used anymore, after I deleted the check